Expecto.Tests/FocusedTests.fs

```diff
@@ -0,0 +1,77 @@
+module Expecto.Focused
```

Comment:
Fine with me, I was using a namespace before, then removed and forgot about the identation. Is better without *=*